server/upstream/upstream-packages-match.txt

```diff
@@ -1130,6 +1140,7 @@ xf86dga:
 xf86-input-evdev:
 xf86-input-joystick:
 xf86-input-keyboard:
+xf86-input-libinput
```

Comment:
Btw, I guess it needs to be added to the other file because it's not hosted on the gnome servers?